sdk/inc/azure/iot/az_iot_hub_client.h

```diff
@@ -443,6 +453,96 @@ AZ_NODISCARD az_result az_iot_hub_client_methods_response_get_publish_topic(
     size_t mqtt_topic_size,
     size_t* out_mqtt_topic_length);
 
+/*
+ *
+ * Commands APIs
+ *
+ */
+
+/**
+ * @brief The MQTT topic filter to subscribe to command requests.
```

Comment:
Yup, I'm tracking this.  There's also a general discussion of whether the legacy twin/method/etc. should point to the PnP flavors in documentation.  *For now* I'm going to leave code as-is until we resolve this, since we maybe looking cross-SDK.